packages/@okta/vuepress-site/docs/guides/pwd-optional-widget-sign-in-email/main/java/integrationsummary.md

```diff
@@ -2,4 +2,8 @@
 
 ![Sequence diagram illustrating the Sign-In Widget's password-optional sign-in use case.](/img/pwd-optional/pwd-optional-java-widget-sign-in-summary.png)
```

Comment:
Change text in "Display verify with an email page" so it sounds less awkward.

Reply:
The Widget calls it "Verify with your email" so change it to that?